src/main/java/cart/controller/MemberController.java

```diff
@@ -0,0 +1,30 @@
+package cart.controller;
+
+import cart.dto.MemberDto;
+import cart.service.MemberService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
```

Comment:
```suggestion
@RestController("/members")
```
이렇게 하지 않고 메서드마다 설정한 이유는 따로 있을까요~?

Reply:
`@requestMapping`같은 것으로 위에서 공통적인 url을 묶지 않은 것은
그냥 개인적으로 메서드마다 url이 있는것이 읽기 편해서였습니다.
까먹으면 위로 올라가서 공통 url을 확인하는게 저는 더 확인하기 불편하더라고요;;